### PR TITLE
Addsidebar

### PR DESCRIPTION
**Pull request by kathleenkong**
Added the supervisor sidebar to the add employees page and the SLE sidebar to the calendar

### CODE REVIEW

**Comment by kathleenkong** [comment]
RAFL @leebianca @bdeleonardis1

File: client/src/components/Cover/Cover.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  }
+
+  showHistory() {
+    this.setState({ typeOfPending: "history" });
+    console.log(this.state.typeOfPending);
+  }
+
+  render() {
+    let typeOfPending = this.state.typeOfPending;
+    console.log(typeOfPending);
```
Comment (leebianca) [inline client/src/components/Cover/Cover.js]: Make sure to take out the console.logs.

**Comment by leebianca** [comment]
Just take out the console.logs but otherwise, looks good!

**Comment by bdeleonardis1** [comment]
Im just going to cleanup the console.logs for you guys to speed this up

**Pull request merged**